source/clear-linux/tutorials/multi-boot/multi-boot-sles.rst

```diff
@@ -0,0 +1,128 @@
+
+#. Click :guilabel:`Accept`.
+
+#. Follow the remaining prompts to complete the installation of SUSE.
+
+#. At this point, |CL| cannot boot because `Grub`
+   is the default boot loader. Follow these steps to make the |CL|
+   Systemd-Boot the default boot loader and add SUSE as a boot option:
+
+   #. Boot into SUSE.
+
+   #. Log in.
+
+   #. Get root privileges with the following command:
+
+      .. code-block:: console
+
+         $ sudo -s
+
+   #. Locate SUSE’s :file:`grub.cfg` in the :file:`/boot/grub2/` directory
+      and look for the primary SUSE :guilabel:`menuentry` section. The
+      highlighted lines identify the kernel, the :file:`initrd` filenames,
+      the root partition UUID, and the additional parameters used. Use this
+      information to create a new Systemd-Boot entry. See Figure 4.
+
+      .. figure:: figures/multi-boot-sles-4.png
+
+         Figure 4: SUSE: grub.cfg
+
+   #. Copy the kernel and the :file:`initrd` file to the EFI partition.
+
+      .. code-block:: console
+
+         # cp /boot/vmlinuz-4.4.21-69-default /boot/efi
+
+         # cp /boot/initrd-4.4.21-69-default /boot/efi
+
+   #. Create a boot entry for SUSE. The file must at least contain these
+      settings:
+
+      +---------+---------------------------------------+
+      | Setting | Description                           |
+      +=========+=======================================+
+      | title   | Text to show in the boot menu         |
+      +---------+---------------------------------------+
+      | linux   | Linux kernel image                    |
+      +---------+---------------------------------------+
+      | initrd  | initramfs image                       |
+      +---------+---------------------------------------+
+      | options | Options to pass to the EFI program or |
+      |         | kernel boot parameters                |
+      +---------+---------------------------------------+
+
+      See the `systemd boot loader documentation`_ for additional
+      details.
+
+      The *options* parameter must specify the root partition UUID and
+      any additional parameters SUSE requires.
+
+      .. note:: The root partition UUID used below is unique to this example.
+
+         .. code-block:: console
+
+            # cd /boot/efi/loader/entries
+
+            # vi suse.conf
+
+         Add the following lines to the :file:`suse.conf` file:
+
+            .. code-block:: console
+
+               title SUSE Linux Enterprise 12 SP2
+
+               linux /vmlinuz-4.4.21-69-default
+
+               initrd /initrd-4.4.21-69-default
+
+               options root=UUID=b9e25e98-a644-4ac3-b955-ae32800ee350 ro
+               resume=/dev/disk/by-uuid/6a50c032-1c1e-4b4a-b799-ca365bb10dc7
+               splash=silent showopts crashkernel=109M,high
+               crashkernel=72M,low
+
+#. Re-install Systemd-Boot to make it the default boot loader.
+
+   .. code-block:: console
+
+      # bootctl install --path /boot/efi
```

Comment:
Change "# bootctl install --path /boot/efi" to "$ sudo bootctl install --path /boot/efi"

Reply:
Text edits deferred to next phase of updates.